code42.json

```diff
@@ -928,7 +938,7 @@
                     "description": "A risk tag to add to the user",
                     "data_type": "string",
                     "required": true,
-                    "primary": false,
+                    "primary": true,
```

Comment:
This allows us to run `add_highrisk_tag` and `remove_highrisk_tag` directly from the individual tags in the table list

Reply:
Nice!